tests/test_scheduler.py

```diff
@@ -327,10 +350,10 @@ def test_task_not_rescheduled_age_limit(self):
         handler = CompletedJobHandler(self.task_scheduler)
 
         scheduler_opts = SchedulingTaskConfig(delay=0, max_retries=0, max_age=3)
-        task = self.registry.add('mytask', 'git', 'commit', {},
-                                 scheduling_cfg=scheduler_opts)
+        task = self.task_scheduler.registry.add('mytask', 'git', 'commit', {}, scheduling_cfg=scheduler_opts)
```

Comment:
Would you mind to reduce the size of this line? Following the style of the original line the format would be:

```
task = self.task_scheduler.registry.add('mytask', 'git', 'commit', {},
                                        scheduling_cfg=scheduler_opts)
```

I think this is easier to read and to review.

There are other lines like these you changed on this commit too.